test/feeds.test.js

```diff
@@ -132,3 +132,53 @@ describe('test POST /feeds endpoint', () => {
     expect(res.status).toEqual(409);
   });
 });
+
+describe.skip('test DELETE /feeds/:id endpoint', () => {
+  let feedData;
+
+  beforeAll(() => {
+    const user = login('Johannes Kepler', 'user1@example.com');
+    feedData = {
+      author: user.name,
+      url: 'http://telescope200.cdot.systems',
+      user: user.id,
+    };
+  });
+
+  beforeEach(() => {
+    logout();
+    login('Johannes Kepler', 'user1@example.com');
+  });
+
+  afterAll(() => logout());
+
+  it('should respond with a 403 status when not logged in', async () => {
+    let res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
```

Comment:
Everyone of these tests does this same thing at the start, so let's move this to the `beforeEach` above, and also add a line right after this to confirm that your feed got added:

```js
    expect(res.status).toEqual(201);
```

Reply:
>Everyone of these tests does this same thing at the start, so let's move this to the `beforeEach` above

I actually originally had this fetch in `beforeEach()` within my first commit (https://github.com/Seneca-CDOT/telescope/pull/958/commits/210ec80c11c6c76f3974d37799f229fb566499d3):

https://github.com/Seneca-CDOT/telescope/blob/210ec80c11c6c76f3974d37799f229fb566499d3/test/feeds.test.js#L136-L150

However, couldn't this potentially lead to a race condition? As can be seen above, enclosing this call to fetch in the `beforeEach` would necessitate exposing a `feedId` to each test function (L137 in the above code snippet). This could lead to a race condition between the three test functions accessing this thrice-modified variable. Thoughts?

Reply:
The test cases in a `describe` always run one-by-one, no races.  Test files can be run in parallel in multiple processes.

Reply:
Wow, I did not know that!

Requested changes made in https://github.com/Seneca-CDOT/telescope/pull/958/commits/5d56f1221e6c3c1d2d0424bec9051ea9b54161fe